docs/src/main/asciidoc/getting-started-testing.adoc

```diff
@@ -1394,3 +1394,13 @@ For `@QuarkusIntegrationTest` tests that result in launcher the application as a
 This can be used by `QuarkusTestResourceLifecycleManager` that need to launch additional containers that the application will communicate with.
 ====
 
+== Troubleshooting
+
+=== Junit 5 parameterized tests failing on Java 16 and newer that results in a `java.lang.IllegalStateException`
```

Comment:
The heading should be separated from the description

Reply:
Ooops, fixed.